src/connectedk8s/azext_connectedk8s/_clientproxyutils.py

```diff
@@ -43,10 +43,7 @@ def close_subprocess_and_raise_cli_error(proc_subprocess, msg):
 
 
 def check_if_csp_is_running(clientproxy_process):
-    if clientproxy_process.poll() is None:
-        return True
-    else:
-        return False
+    return (clientproxy_process.poll() is None)
```

Comment:
What is the expected return object of clientproxy_process.poll()?

Reply:
Discussed offline